fixture-monkey-api/src/main/java/com/navercorp/fixturemonkey/api/option/GenerateOptionsBuilder.java

```diff
@@ -529,15 +524,10 @@ public GenerateOptions build() {
 			defaultNullInjectGenerator = defaultNullInjectGeneratorOperator.apply(defaultNullInjectGenerator);
 		}
 
-		int defaultArbitraryContainerMaxSize = defaultIfNull(
-			this.defaultArbitraryContainerMaxSize,
-			() -> GenerateOptions.DEFAULT_ARBITRARY_CONTAINER_MAX_SIZE
+		ArbitraryContainerInfoGenerator defaultArbitraryContainerInfoGenerator = defaultIfNull(
+			this.defaultArbitraryContainerInfoGenerator,
+			() -> context -> new ArbitraryContainerInfo(0, 3)
```

Comment:
`3` could be replaced by `GenerateOptions .DEFAULT_ARBITRARY_CONTAINER_MAX_SIZE`

Reply:
In commit e464960, I replaced literal to pre-initialized constant 🙂